ui/app/templates/components/okta-number-challenge.hbs

```diff
@@ -3,7 +3,17 @@
     <div class="field has-top-margin-xs">
       <p data-test-okta-number-challenge-description>
         To finish signing in, you will need to complete an additional MFA step.</p>
-      {{#if this.oktaNumberChallengeCorrectAnswer}}
+      {{#if this.errorThrown}}
+        <div class="has-top-margin-s">
+          <MessageError @errorMessage="There was a problem" />
```

Comment:
Is there not an error returned from the API? If there is then we should return that error instead of a generic message.

Reply:
Yes there's an error returned from the API but that one appears on the main login page after they click the "Return to login" button. The design originally had the error message just to say there was a problem. Should I display the error from the API twice or should it stay this way?

Reply:
Follow the designs. As long as we display the API error somewhere, we should be good. Thanks for following up.